whatis277/harvest
# bean: Add a basic view driver

Basic view driver aligned with clean architecture

It's minimal with no view models
They can be added when needed

Entire idea is to separate concerns

Testing instructions:
1. `dc up bean`
2. Visit [`/`](http://localhost:8080/)
3. Go to [`/get-started`](http://localhost:8080/get-started)
4. Enter an email and "login"
5. Ensure you get a message indicating an email was sent

![image](https://github.com/whatis277/harvest/assets/5631091/8b2f173d-85f3-4d8a-8385-824695d3bb37)

![image](https://github.com/whatis277/harvest/assets/5631091/d421af3c-b907-435a-8122-eba5a11a4621)
